addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.ChannelMemberListCategory" owl="1">
+        <h6 class="m-2"><t t-esc="title"/> - <t t-esc="members.length"/></h6>
```

Comment:
maybe merge these two properties and make it translatable

Reply:
Change done